perceval/algorithm/tomography/tomography.py

```diff
@@ -107,81 +106,65 @@ class StateTomography(AAlgorithm):
     - Computes parameters required to do state tomography
     - Performs Tomography experiment - Computes and Returns density matrices for each input state
     """
-    def __init__(self, nqubit: int, operator_processor: Processor, post_process=False, renormalization=None):
+    def __init__(self, operator_processor: Processor):
         super().__init__(processor=operator_processor)
-        self._nqubit = nqubit
         self._operator_processor = operator_processor  # Gate operation under study
-        self._backend = operator_processor.backend  # default - SLOSBackend()
-        self._source = operator_processor.source  # default - ideal source
-        self._post_process = post_process
-        self._renormalization = renormalization
-        self._heralded_modes = [(key, value) for key, value in operator_processor.heralds.items()]
-        self._size_hilbert = 2 ** nqubit
-        # Todo: I need to put this number somewhere, i forgot. find [number of elements = between (0 -> 4**nqubits-1)]
-
-    def _input_state_dist_config(self):
-        # Configures the input state for the Processor
-        input_state = BasicState("|1,0>")
-        for _ in range(1, self._nqubit):
-            # setting the input state for the gate qubit modes
-            input_state *= BasicState("|1,0>")
-        for m in self._heralded_modes:
-            # setting the input for heralded modes of the given processor
-            input_state *= BasicState([m[1]])
-        input_distribution = self._source.generate_distribution(expected_input=input_state)
-        return input_distribution
+        self._nqubit = int(operator_processor.m / 2)  # nqubit
+        self._size_hilbert = 2 ** self._nqubit
+        self._gate_logical_perf = None
 
-    def _compute_probs(self, prep_state_indices, meas_pauli_basis_indices):
+    def _configure_processor(self, prep_state_indices, meas_pauli_basis_indices):
         # Adds preparation and measurement circuit to input processor (with the gate operation under study)
```

Comment:
Can you use the docstrings convention ? Like that:
```
        """_summary_

        :param prep_state_indices: _description_
        :param meas_pauli_basis_indices: _description_
        :return: _description_
        """
```
And maybe in other places too ?

Reply:
I will update the docstring today :-)